.github/workflows/tests.yml

```diff
@@ -37,8 +29,18 @@ jobs:
             - name: Install dependencies
               run: poetry install
 
-            - name: Run tests
-              run: poetry run pytest -v
+            # Run a subset of tests first which run quickly to fail fast.
+            - name: Run fast unit tests
+              run: poetry run pytest -v tests/anoph
```

Comment:
Pulling out a subset of tests here which are guaranteed to run fast, so we fail faster if there are any obvious bugs.